tools/unittestharness.py

```diff
@@ -99,22 +174,40 @@ def run(self):
         else:
             print("    Failures: %d; tests = %s" % (failcount, failtests))
 
+        if self.xml_outfile != '':
+            fd = open(f'{self.cwd}/{self.xml_outfile}', 'w')
```

Comment:
In general you should use `os.path.join` to concatenate directories. It deals e.g. with the case where `self.xml_outfile` is an absolute path (starting with '/'). Similarly if the first part is empty (not possible) here)

Reply:
Implemented via 9ca67383ce5520b516c0cf1a31f139cbf54160aa